src/com/gmail/chernii/oleksii/FlowerStore.java

```diff
@@ -0,0 +1,77 @@
+package com.gmail.chernii.oleksii;
```

Comment:
Let's change the root package from com.gmail.chernii.oleksii; to com.my.flowers (or think of your own version)
And let's put all flower classes into an entity or domain package.
It would be com.my.flowers.domain (or com.my.flowers.entity)

Reply:
root package = com.my.flowers